apps/clock/index.html

```diff
@@ -74,7 +75,5 @@
       <li role="presentation"><a id="timer-tab" aria-controls="timer-panel" aria-selected="false" role="tab" href="#timer-panel" data-l10n-id="timer">Timer</a></li>
       <li role="presentation"><a id="stopwatch-tab" aria-controls="stopwatch-panel" aria-selected="false" role="tab" href="#stopwatch-panel" data-l10n-id="stopwatch">Stopwatch</a></li>
     </ul>
```

Comment:
what about using this PR to also remove the `Alarm`, `Timer` and `Stopwatch` strings from the markup? - don't know what is your opinion about that but I'm seen a lot of PRs lately removing the English strings from the HTML to avoid mistakes